ios/Classes/TiAirshipModule.m

```diff
@@ -1,56 +1,58 @@
-/* Copyright Urban Airship and Contributors */
+/* Copyright Airship and Contributors */
 
-#import "ComUrbanairshipModule.h"
+#import "TiAirshipModule.h"
 #import "TiBase.h"
 #import "TiHost.h"
 #import "TiUtils.h"
-#import "AirshipLib.h"
-#import "UAAssociatedIdentifiers.h"
-#import "ComUrbanairshipDeeplink.h"
 
-@interface ComUrbanairshipModule()
-@property (nonatomic, copy) NSDictionary *launchPush;
-@property (nonatomic,copy) NSString *deepLink;
+NS_ASSUME_NONNULL_BEGIN
+
+@interface TiAirshipModule()
+@property (nonatomic, copy, nullable) NSDictionary *launchPush;
+@property (nonatomic, copy, nullable) NSString *deepLink;
 @end
 
-@implementation ComUrbanairshipModule
+@implementation TiAirshipModule
 
 #pragma mark Internal
 
-// this is generated for your module, please do not change it
--(id)moduleGUID {
-	return @"fbe797b5-475d-456a-bbb4-9bb9919fd63f";
+// This is generated for your module, please do not change it
+- (id)moduleGUID {
+    return @"240dc468-ccad-479d-9510-14e9a7cae5c9";
```

Comment:
I think this was previously incorrect, set me straight if I'm wrong

Reply:
i think its supposed to match